### PR TITLE
[NFC] Code cleanup revealed by self-build.

### PR DESCRIPTION
**Pull request by zahiraam**

### CODE REVIEW

File: llvm-spirv/lib/SPIRV/SPIRVToOCL.cpp [hunk 1]
```diff
@@ -537,8 +537,7 @@ llvm::createSPIRVBIsLoweringPass(Module &M,
   case SPIRV::BIsRepresentation::SPIRVFriendlyIR:
     // nothing to do, already done
     return nullptr;
-  default:
-    llvm_unreachable("Unsupported built-ins representation");
-    return nullptr;
```
Comment (AlexeySachkov) [inline llvm-spirv/lib/SPIRV/SPIRVToOCL.cpp:542]: This change should be upstreamed to KhronosGroup/SPIRV-LLVM-Translator GitHub repo

**Review by kbobrovs — APPROVED** [review]
LowerESIMD.cpp LGTM

**Review by DenisBakhvalov — DISMISSED** [review]
ESIMD part looks good.

File: llvm-spirv/lib/SPIRV/SPIRVToOCL.cpp (hunk 1 above)
Comment (zahiraam) [reply llvm-spirv/lib/SPIRV/SPIRVToOCL.cpp:542]: Do I need to revert this or leave it and upstream to Khronos repo too?

Comment (zahiraam) [reply llvm-spirv/lib/SPIRV/SPIRVToOCL.cpp]: Reverted those files. Thanks.

**Review by v-klochkov — APPROVED** [review]

**Review by kbobrovs — APPROVED** [review]

**Pull request merged**